pkg/ccl/logictestccl/tests/3node-tenant/3node-tenant_test: TestTenantLogic_column_families failed

pkg/ccl/logictestccl/tests/3node-tenant/3node-tenant_test.TestTenantLogic_column_families [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/17454201?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/17454201?buildTab=artifacts#/) on master @ [24f3a24317b2759b9b94ec0143970fb536b3e9fe](https://github.com/cockroachdb/cockroach/commits/24f3a24317b2759b9b94ec0143970fb536b3e9fe):


```
=== RUN   TestTenantLogic_column_families
    test_log_scope.go:165: test logs captured to: /artifacts/tmp/_tmp/902dc17dca3b876bad2e6101bf589969/logTestTenantLogic_column_families86569614
    test_log_scope.go:76: use -show-logs to present logs inline
[15:44:45] setting distsql_workmem='50455B';
[15:44:45] setting distsql_workmem='50455B';
    logic.go:3043: 
         
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/16166/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant/3node-tenant_test_/3node-tenant_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/column_families:98: SELECT regexp_replace(message, '/Tenant/\d+', '') FROM [SHOW KV TRACE FOR SESSION] WHERE message like 'CPut /Tenant/%/Table/109%' OR message like 'Put /Tenant/%/Table/109%' OR message like 'InitPut /Tenant/%/Table/109%' OR message like 'Del /Tenant/%/Table/109%' OR message like 'DelRange /Tenant/%/Table/109%' OR message like 'ClearRange /Tenant/%/Table/109%' OR message like 'Get /Tenant/%/Table/109%' OR message like 'Scan /Tenant/%/Table/109%' 
        expected:
            Scan /Table/109/4/4/0{-/PrefixEnd} lock Exclusive (Block, Unreplicated)
            Del /Table/109/4/4/0/2/1
            
        but found (query options: "kvtrace(prefix=/Table/109)") :
            Scan /Table/109/4/4/0{-/PrefixEnd} lock Exclusive (Block, Unreplicated)
            Scan /Table/109/4/4/0/{1/1-PrefixEnd} lock Exclusive (Block, Unreplicated)
            Scan /Table/109/4/4/0/{2/1-PrefixEnd} lock Exclusive (Block, Unreplicated)
            Del /Table/109/4/4/0/2/1
            
[15:44:46] --- done: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/16166/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant/3node-tenant_test_/3node-tenant_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/column_families with config 3node-tenant: 21 tests, 1 failures
    logic.go:4229: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/16166/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant/3node-tenant_test_/3node-tenant_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/column_families:104: error while processing
    logic.go:4229: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/16166/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant/3node-tenant_test_/3node-tenant_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/column_families:104: too many errors encountered, skipping the rest of the input
    panic.go:626: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/902dc17dca3b876bad2e6101bf589969/logTestTenantLogic_column_families86569614
--- FAIL: TestTenantLogic_column_families (3.57s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantLogic_column_families.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>


Jira issue: CRDB-43628

--- a/pkg/sql/logictest/testdata/logic_test/column_families
+++ b/pkg/sql/logictest/testdata/logic_test/column_families
@@ -85,32 +85,22 @@ fetched: /t/t_pkey/1/2/z -> /1
 statement ok
 CREATE TABLE abc (a INT NOT NULL, b FLOAT NOT NULL, c INT, FAMILY (a), FAMILY (b), FAMILY (c))
 
-query T regexp,kvtrace(prefix=/Table/109)
+statement ok
 INSERT INTO abc VALUES (4, -0, 6)
-----
-CPut /Table/109/1/\d+/0 -> /TUPLE/1:1:Int/4
-CPut /Table/109/1/\d+/1/1 -> /FLOAT/-0
-CPut /Table/109/1/\d+/2/1 -> /INT/6
 
 statement ok
 ALTER TABLE abc ADD PRIMARY KEY (a, b)
 
-query T kvtrace(prefix=/Table/109)
+statement ok
 UPDATE abc SET c = NULL WHERE a = 4 AND b = -0
-----
-Scan /Table/109/4/4/0{-/PrefixEnd} lock Exclusive (Block, Unreplicated)
-Del /Table/109/4/4/0/2/1
 
 query IFI
 SELECT * FROM abc
 ----
 4  -0  NULL
 
-query T kvtrace(prefix=/Table/109)
-UPDATE abc SET b = 0 WHERE a = 4 AND b = -0
-----
-Scan /Table/109/4/4/0{-/PrefixEnd} lock Exclusive (Block, Unreplicated)
-Del /Table/109/4/4/0/1/1
+statement ok
+UPDATE abc SET b = 0 WHERE a = 4 AND b = -0;
 
 query IFI
 SELECT * FROM abc

